systems/common/pb/ukama/subscriber.proto

```diff
@@ -21,7 +20,7 @@ import "google/protobuf/wrappers.proto";
     string lastName = 3 [json_name = "last_name"];
     string subscriberId = 4 [(validator.field) = {uuid_ver: 4, string_not_empty: true}, json_name = "subscriber_id"];
     string networkId = 5 [(validator.field) = {uuid_ver: 4, string_not_empty: true}, json_name = "network_id"];
-    string email  = 6;
+    string email  = 6 [(validator.field) = { human_error:"must be an email format" regex: "^$|^[a-z0-9._%+\\-]+@[a-z0-9.\\-]+\\.[a-z]{2,4}$"}];
```

Comment:
You can implement email validation at the system level. We've decided to use all validations from common.

Reply:
Please open an issue on it, This need to be address on all systems. We can address it in separate PR.